src/runtimes/node/in_source_config/index.ts

```diff
@@ -61,6 +62,25 @@ export const findISCDeclarationsInPath = async (
   return findISCDeclarations(source, { functionName, featureFlags, logger })
 }
 
+/**
+ * Normalizes method names into arrays of uppercase strings.
+ * (e.g. "get" becomes ["GET"])
+ */
+const normalizeMethods = (method: unknown, name: string): string[] | undefined => {
+  const methods = Array.isArray(method) ? method : [method]
+
+  // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
Oof! Do we have to name both variables `method`?

Reply:
no we don't ;D [ecb8e6a](https://github.com/netlify/zip-it-and-ship-it/pull/1539/commits/ecb8e6aea10c72901e3788e21a1e1dc63d617637)